MCprep_addon/materials/prep.py

```diff
@@ -35,171 +36,158 @@
 			"the active texture pack and using normal/spec passes"),
 		default=1.0,
 		max=1,
 		min=0)
-	makeSolid: bpy.props.BoolProperty(
+	makeSolid: BoolProperty(
 		name="Make all materials solid",
 		description="Make all materials solid only, for shadows and rendering",
 		default=False)
-	syncMaterials: bpy.props.BoolProperty(
+	syncMaterials: BoolProperty(
 		name="Sync materials",
 		description=(
 			"Synchronize materials with those in the active "
 			"pack's materials.blend file"),
 		default=True)
-	# newDefault: bpy.props.BoolProperty(
+	# newDefault: BoolProperty(
 	#	name="Use custom default material",
 	#	description="Use a custom default material if you have one set up",
 	#	default=False)
-	packFormat: bpy.props.EnumProperty(
+	packFormat: EnumProperty(
 		name="Pack Format",
 		description="Change the pack format when using a PBR resource pack.",
 		items=pack_formats
 	)
-	useEmission: bpy.props.BoolProperty(
+	useEmission: BoolProperty(
 		name="Use Emission",
 		description="Make emmisive materials emit light",
 		default=True
 	)
 
-
-def draw_mats_common(self, context: Context) -> None:
-	row = self.layout.row()
-	col = row.column()
-	engine = context.scene.render.engine
-	if engine == 'CYCLES' or engine == 'BLENDER_EEVEE':
-		col.prop(self, "packFormat")
-		col.prop(self, "usePrincipledShader")
-	col.prop(self, "useReflections")
-	col.prop(self, "makeSolid")
-
-	anim_row = col.row()
-	can_swap_text = util.is_atlas_export(context)
-	anim_row.enabled = can_swap_text
-	if can_swap_text:
-		anim_row.prop(self, "animateTextures")
-	else:
-		anim_row.prop(self, "animateTextures", text="Animate textures (disabled due to import settings)")
-	col.prop(self, "autoFindMissingTextures")
-
-	row = self.layout.row()
-	row.prop(self, "useExtraMaps")
-	row.prop(self, "syncMaterials")
-	row = self.layout.row()
-	# row.prop(self, "newDefault")
-
-	# col = row.column()
-	# col.prop(self, "normalIntensity", slider=True)
-
-	row = self.layout.row()
-	col = row.column()
-	col.prop(self, "improveUiSettings")
-	col = row.column()
-	col.prop(self, "combineMaterials")
-	row = self.layout.row()
-	row.prop(self, "optimizeScene")
-	row.prop(self, "useEmission")
-
-
-class MCPREP_OT_prep_materials(bpy.types.Operator, McprepMaterialProps):
-	"""Fixes materials and textures on selected objects for Minecraft rendering"""
-	bl_idname = "mcprep.prep_materials"
-	bl_label = "MCprep Materials"
-	bl_options = {'REGISTER', 'UNDO'}
-
-	skipUsage: bpy.props.BoolProperty(
-		default=False,
-		options={'HIDDEN'})
-
-	def invoke(self, context, event):
-		return context.window_manager.invoke_props_dialog(
-			self, width=300 * util.ui_scale())
-
-	def draw(self, context):
-		draw_mats_common(self, context)
-
-	track_function = "materials"
-	track_param = None
-	track_exporter = None
-	@tracking.report_error
-	def execute(self, context):
-
+	def draw_before(self, context: Context) -> None:
+		pass
+	
+	def draw_after(self, context: Context) -> None:
+		pass
```

Comment:
Don't leave functions that serve no purpose.

Reply:
I do agree with this point to make sure we don't leave leftover code, presuming this was WIP or something.